packages/sdk/dappkit/README.md

```diff
@@ -195,56 +196,58 @@ requestTxSig(
       from: this.state.address,
       to: stableToken.contract.options.address,
```

Comment:
This still causes compile errors for me -- is there a reason not to use `stableToken.address` (same for the other contract addresses)?